Pokedex_ios/View/Cells/PokemonCell/PokemonTableViewCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  PokemonTableViewCell.swift
+//  Pokedex_ios
+//
+//  Created by admin on 6/29/22.
+//
+
+import UIKit
+
+
+class PokemonTableViewCell: UITableViewCell {
+    
```

Comment:
Add the identifier and uiNibName as static fields, since they are going to be called using the class name